Library/Homebrew/cmd/audit.rb

```diff
@@ -186,6 +186,7 @@ def audit_file
       [/^  keg_only/,                      "keg_only"],
       [/^  option/,                        "option"],
       [/^  depends_on/,                    "depends_on"],
+      [/^  (go_resource|resource)/,        "resources"],
```

Comment:
Sure, that's more than fine with me! Wasn't sure if people wanted explicit or neat, so went for the former.

I'll address it later today; want to wait a bit to see if any additional review occurs before chucking another build on the queue.